modules/ROOT/pages/get-started.adoc

```diff
@@ -0,0 +1,143 @@
+= Getting Started with Partner Manager
+
+Partner Manager offers a simplified, configuration-driven B2B integration solution to manage B2B trading partner ecosystems. To use Partner Manager, follow these steps:
+
+. <<assign-permissions,Assign permissions to Partner Manager users>>.
+. <<set-up-env,Set up the Partner Manager environment>>.
+. <<payload-storage-api,Set up the Payload Storage API>>.
+. <<configure-host-profile,Configure a host profile>>.
+. <<configure-partners,Configure a profile for each of your trading partners>>.
+. <<configure-third-party,Optionally configure third party connections>>.
+. <<configure-message-flows,Configure message flows>>.
+. <<deploy-test-message-flows,Deploy and test the configured message flows>>.
+. <<monitor-transmissions,Monitor transmissions and individual messages>>.
+. <<troubleshoot-acks,Troubleshoot rejected or overdue acknowledgements>>.
+
+[[assign-permissions]]
+== Assign Permissions to Partner Manager Users
+
+For each sandbox and production environment, use Anypoint Access Management to assign permissions to each Partner Manager user. You can assign the following permissions:
```

Comment:
where in Access Management?

Reply:
I had talked to Sara about expanding the AIM permissions and reusing them. 

Here's what I did for MQ:
https://docs.mulesoft.com/mq/mq-access-management#anypoint-mq-permissions

Reply:
I can talk to Sara about the text for Partner Manager permissions in the Access Management doc. I also noticed there is a bug in the Access Management software that repeats each permission bullet.

As for where in Access Management, it is under Users, which makes sense sense it is user permissions. I don't think this needs to be called out.